tests/console/yast2_proxy.pm

```diff
@@ -13,6 +13,38 @@
 use strict;
 use base "console_yasttest";
 use testapi;
+use utils "type_string_slow";
+
+my %sub_menu_needles = (
+    start_up      => 'yast2_proxy_start-up',
+    http_ports    => 'yast2_proxy_http_ports_selected',
+    patterns      => 'yast2_proxy_http_refresh_patterns_selected',
+    cache_setting => 'yast2_proxy_http_cache_setting_selected',
+    cache_dir     => 'yast2_proxy_http_cache_directory_selected',
+    access_ctrl   => 'yast2_proxy_http_access_control_selected',
+    log_timeouts  => 'yast2_proxy_logging_timeouts_selected',
+    miscellanous  => 'yast2_proxy_miscellanous_selected'
+);
+
+sub select_sub_menu {
+    my ($initial_screen, $wanted_screen) = @_;
+    send_key_until_needlematch $sub_menu_needles{$initial_screen}, 'tab';
+    wait_still_screen 1;
+    send_key 'down';
+    assert_screen $sub_menu_needles{$wanted_screen};
+    wait_screen_change { send_key 'ret', 1; };
```

Comment:
In general I'm with you here, but for these tests on leap, it improves stability. Wait_screen_change doesn't help in most of the cases because change touches less than 3-5% of the screen, so it returns immediately.